<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="base_button">
        <property name="text">
         <string>Base</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="farcitura_button">
        <property name="text">
         <string>Farcitura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="other_button">
        <property name="text">
         <string>Altre richieste</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_1">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="styleSheet">
          <string notr="true">border:none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QFrame" name="frame_7">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>American Typewriter</family>
                 <pointsize>25</pointsize>
                </font>
               </property>
               <property name="text">
                <string>SCEGLI IL TIPO DI BASE</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item alignment="Qt::AlignVCenter">
           <widget class="QFrame" name="frame_8">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>American Typewriter</family>
              <pointsize>17</pointsize>
             </font>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QRadioButton" name="radioButton_2">
               <property name="font">
                <font>
                 <family>American Typewriter</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Pan di spagna</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QRadioButton" name="radioButton">
               <property name="font">
                <font>
                 <family>American Typewriter</family>
                 <pointsize>17</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Millefoglie</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignTop">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>American Typewriter</family>
              <pointsize>25</pointsize>
             </font>
            </property>
            <property name="text">
             <string>SCEGLI LA FARCITURA</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignVCenter">
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Crema</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Cioccolata</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="text">
             <string>Crema Chantillie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_8">
            <property name="text">
             <string>Altro</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QFrame" name="frame_12">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QCheckBox" name="checkBox_9">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>84</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Panna</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>50</y>
            <width>84</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Frutta</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_11">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>50</y>
            <width>84</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Candeline</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_13">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_14">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>100</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Scrivi qui le tue richieste</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>Ordina</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
